man/systemd-gpt-auto-generator.xml

```diff
@@ -187,6 +187,12 @@
     <filename>/dev/mapper/home</filename> and <filename>/dev/mapper/srv</filename>. Note that this might
     create conflicts if the same partition is listed in <filename>/etc/crypttab</filename> with a different
     device mapper device name.</para>
+    
+    <para>When systemd is running in the initrd the <filename>/</filename> partition may be encrypted in LUKS
+      format as well. In this case, a device mapper device is set up under the name <filename>/dev/mapper/root</filename>,
```

Comment:
Any reason why you didn't use the same indentation style as the surrounding paras?

Reply:
Probably because my vim auto-indent kicked in.